### ISSUE TITLE
[SF][ToolbarButton][A11y]:[Missing accessibleRole prop for ToolbarButton]

### ISSUE THREAD

**Issue by tandan0813** [UI5/webcomponents#13070]
### Bug Description

Can't set accessibleRole for the ToolbarButton, if the button works like a link(navigate to another page), then it will cause A11y issue.

### Expected Behaviour

Can set accessibleRole = "link" for a ToolbarButton

### Priority

None

### UI5 Web Components Version

2.19.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by s-todorova** [comment]
Hello @UI5/ui5-team-ctr-pir,

Could you please take a look at this feature request about providing accessibleRole for the ToolbarButton?

Best,
Siyana

**Comment by judith9209** [comment]
SAPDESIGNSYSTEM-458
